src/components/DocumentContainer/DocumentContainer.js

```diff
@@ -202,7 +202,7 @@ class DocumentContainer extends React.PureComponent {
 
     return (
       <div className={className} ref={this.container} data-element="documentContainer" onScroll={this.handleScroll} onTransitionEnd={this.onTransitionEnd}>
-        <div className="document" ref={this.document}></div>
+        <div className="document" ref={this.document} tabIndex="-1"></div>
```

Comment:
Why do we need a `tabIndex`?

Reply:
When using the up/down arrow buttons, I was noticing that the document would randomly lose focus. So if you have two page showing

- in non continuous mode as soon as the pages change, the up and down buttons stop working till you click on the document again
- in continuous mode, after scrolling pass a page with the arrow keys, it'll randomly lose focus and but button stops working

The `tabIndex` allow the document element to be focus using JavaScript. It seem some other code (probably in our core) is now able to keep focus on the document after adding this. I was trying stuff like 
```
setTimeout(() => {element.focus()} ,0)
```
but it turned out adding `tabIndex` was all I needed. I'll test this a bit more in Firefox, Edge, and IE to make sure this fix works